Packs/Campaign/Scripts/GetCampaignIncidentsInfo/GetCampaignIncidentsInfo.py

```diff
@@ -78,6 +78,10 @@ def convert_incident_to_hr(incident):
         if key == 'severity':
             converted_incident[key] = SEVERITIES.get(converted_incident.get(key), 'None')
 
+        if key == 'similarity':
+            converted_incident[key] = str(converted_incident[key])[:4] if str(converted_incident[key])[0] != '1' \
```

Comment:
can you remind me what was the problem with something like `round(x, 2)`?

Reply:
done.